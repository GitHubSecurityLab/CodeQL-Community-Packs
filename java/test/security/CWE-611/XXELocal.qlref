security/CWE-611/XXELocal.ql
