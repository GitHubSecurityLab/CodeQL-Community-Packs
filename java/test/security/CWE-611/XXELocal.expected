edges
| XXELocal.java:15:39:15:63 | new FileInputStream(...) : FileInputStream | XXELocal.java:16:51:16:61 | inputStream : FileInputStream | provenance | Src:MaD:42998  |
| XXELocal.java:16:35:16:62 | new InputSource(...) : InputSource | XXELocal.java:24:25:24:35 | inputSource | provenance |  |
| XXELocal.java:16:51:16:61 | inputStream : FileInputStream | XXELocal.java:16:35:16:62 | new InputSource(...) : InputSource | provenance | MaD:49767 |
nodes
| XXELocal.java:15:39:15:63 | new FileInputStream(...) : FileInputStream | semmle.label | new FileInputStream(...) : FileInputStream |
| XXELocal.java:16:35:16:62 | new InputSource(...) : InputSource | semmle.label | new InputSource(...) : InputSource |
| XXELocal.java:16:51:16:61 | inputStream : FileInputStream | semmle.label | inputStream : FileInputStream |
| XXELocal.java:24:25:24:35 | inputSource | semmle.label | inputSource |
subpaths
#select
| XXELocal.java:24:25:24:35 | inputSource | XXELocal.java:15:39:15:63 | new FileInputStream(...) : FileInputStream | XXELocal.java:24:25:24:35 | inputSource | Unsafe parsing of XML file from $@. | XXELocal.java:15:39:15:63 | new FileInputStream(...) | user input |
